src/doc/unstable-book/src/compiler-flags/print-crate-root-lint-levels.md

```diff
@@ -0,0 +1,23 @@
+# `print=crate-root-lint-levels`
+
+The tracking issue for this feature is: [#139180](https://github.com/rust-lang/rust/issues/139180).
+
+------------------------
+
+This option of the `--print` flag print the list of lints with print out all the lints and their associated levels (`allow`, `warn`, `deny`, `forbid`) based on the regular Rust rules at crate root, that is *(roughly)*:
```

Comment:
Discussion: maybe hammer in the point that this is *not necessarily* equivalent to the *finalized* lint level at a given lexical construct position in a program?

Reply:
The print request should always reflect the actual finalized lint level at crate root, the name of print request mentions "crate root" and the documentation also mention it as well.

I'm not sure what else I could do to make it even more explicit.

Reply:
That's fair.